### PR TITLE
JSON API: Add Sync Endpoint

### PR DESCRIPTION
**Pull request by lezama**
Add POST /sites/%s/sync endpoint. Schedules a full sync. cc @enejb

### CODE REVIEW

**Comment by lezama** [comment]
image

File: json-endpoints/jetpack/json-api-jetpack-endpoints.php [hunk 1]
```diff
@@ -562,3 +562,19 @@
...
+	'method'          => 'POST',
+	'path'            => '/sites/%s/sync',
+	'stat'            => 'sync',
+	'path_labels' => array(
+		'$site' => '(int|string) The site ID, The site domain'
+	),
+	'response_format' => array(
+		'scheduled' => '(bool) Whether or not the synchronisation was scheduled'
+	),
+	'example_request' => 'https://public-api.wordpress.com/rest/v1/sites/example.wordpress.org/sync'
```
Comment (enejb) [inline json-endpoints/jetpack/json-api-jetpack-endpoints.php]: Should the version be 1 or 1.1?

**Comment by roccotripaldi** [comment]
Code looks great!

File: json-endpoints/jetpack/json-api-jetpack-endpoints.php (hunk 1 above)
Comment (beaulebens) [inline json-endpoints/jetpack/json-api-jetpack-endpoints.php]: Definitely should be 1.1.

**Comment by roccotripaldi** [comment]
screen shot 2015-03-10 at 6 46 01 pm

**Comment by lezama** [comment]
Thanks for the Review!

**Pull request merged**